README.md

```diff
@@ -1,2 +1,262 @@
-# Sheetsu Web Client
+# Connect database on the front-end with only HTML
+# Spreadsheet as a front-end database
+# Spreadsheet as a JavaScript database
 
+Connect database to a website without any backend. Operating in a static HTML, hosted anywhere, easy to use by anyone, works everywhere in any browser.
+Google Spreadsheets being interface and database. 
+Adds a lot of simplicity and allow non-IT people to use the database, make changes and analyze data.
```

Comment:
What about

```
Connect database to a website without any backend, straight from HTML.

Works everywhere. Database hosted as a Google Spreadsheets.
- Speed up development
- Add simplicity
- Allow non-IT people to play with data
```

I think it's cleaner explanation and makes more sense. Also less fluff talk.